src/network/constants.rs

```diff
@@ -42,6 +42,20 @@ use std::{fmt, io, ops};
 use consensus::encode::{self, Encodable, Decodable};
 
 /// Version of the protocol as appearing in network message headers
+/// This constant is used to signal to other peers which features you support.
+/// Increasing it implies that your software also supports every feature prior to this version.
+/// Doing so without support may lead to you incorrectly banning other peers or other peers banning you.
+/// These are the features required for each version:
+/// 70016 - Support receiving messages between `version` and `verack` message
```

Comment:
It was pointed out to me that this only implies the wtxidrelay message - no other messages are allowed between version and verack.